### PR TITLE
Mynewt 1.8.0

### PR DESCRIPTION
**Pull request by szymon-czapracki**

### CODE REVIEW

**Comment by szymon-czapracki** [comment]
@ncasaril FYI

**Comment by ncasaril** [comment]
Would those packages that today use newtmgr have to change to use smp? For instance: 

- sys/uwbcfg
- mgmt/bcast_ota
- lib/nmgr_uwb
- lib/nmgr_cmds

File: apps/twr_tag_tdma/pkg.yml [hunk 1]
```diff
@@ -32,9 +32,11 @@ pkg.deps:
     - "@apache-mynewt-core/sys/shell"
     - "@apache-mynewt-core/sys/log/full"
     - "@apache-mynewt-core/sys/stats/full"
-#    - "@apache-mynewt-core/encoding/json"
+#   - "@apache-mynewt-core/encoding/json"
 #   - "@apache-mynewt-core/mgmt/imgmgr"
-    - "@apache-mynewt-core/mgmt/newtmgr"
+#   - "@apache-mynewt-core/mgmt/newtmgr"
```
Comment (ncasaril) [inline apps/twr_tag_tdma/pkg.yml:37]: Don't leave the old newtmgr here as a comment, just remove it.

File: apps/syscfg/pkg.yml [hunk 2]
```diff
@@ -35,7 +35,9 @@ pkg.deps:
     - "@apache-mynewt-core/sys/log/full"
     - "@apache-mynewt-core/sys/stats/full"
 #   - "@apache-mynewt-core/mgmt/imgmgr"
-    - "@apache-mynewt-core/mgmt/newtmgr"
+#   - "@apache-mynewt-core/mgmt/newtmgr"
```
Comment (ncasaril) [inline apps/syscfg/pkg.yml:38]: Don't leave the old newtmgr here as a comment, just remove it.

File: apps/twr_tag_tdma/pkg.yml (hunk 1 above)
Comment (szymon-czapracki) [reply apps/twr_tag_tdma/pkg.yml]: Done

File: apps/syscfg/pkg.yml (hunk 2 above)
Comment (szymon-czapracki) [reply apps/syscfg/pkg.yml]: Done

File: lib/smp_uwb/syscfg.yml [hunk 3]
```diff
@@ -1,12 +1,12 @@
 
 syscfg.defs:
-    NMGR_UWB_ENABLED:
+    SMP_UWB_ENABLED:
         description: 'Enable UWB transport for NewtMgr'
```
[the remark this answers is not in the document]
Comment (szymon-czapracki) [reply lib/smp_uwb/syscfg.yml]: Done

**Comment by ncasaril** [comment]
This only fails because the travis build assumes mynewt 1.7.

**Pull request closed without merging**